cmd/snap/cmd_snap_op.go

```diff
@@ -1209,6 +1230,41 @@ func (x *cmdRefresh) Execute([]string) error {
 	return x.refreshMany(names, opts)
 }
 
+func (x *cmdRefresh) TrackRefreshes() (err error) {
+	names := installedSnapNames(x.Positional.Snaps)
+
+	snaps, err := x.client.List(names, nil)
+	if err != nil {
+		return err
+	}
+
+	sort.Sort(snapsByName(snaps))
+
+	config := snapConfig{
+		Snaps: make(map[string]snapChannel),
+	}
```

Comment:
With the output struct defined inline, this can become:

```suggestion
	config := make(map[string]snapChannel, len(snaps))
```